src/pcapi/routes/native/v1/offers.py

```diff
@@ -44,3 +46,12 @@ def send_offer_webapp_link(user: User, offer_id: int) -> None:
     """
     offer = Offer.query.options(joinedload(Offer.venue)).filter(Offer.id == offer_id).first_or_404()
     send_user_webapp_offer_link_email(user, offer)
+
+
+@blueprint.native_v1.route("/send_offer_link_by_push/<int:offer_id>", methods=["POST"])
+@spectree_serialize(on_success_status=204, api=blueprint.api)  # type: ignore
+@authenticated_user_required
+def send_offer_link_by_push(user: User, offer_id: int) -> None:
+    offer = Offer.query.filter(Offer.id == offer_id).first_or_404()
```

Comment:
Il n'y a pas de raison d'avoir plusieurs offres sur un id.
```suggestion
    offer = Offer.query.filter(Offer.id == offer_id).get_or_404()
```